plugins/heroku/.toolkitrc.yml

```diff
@@ -11,4 +11,13 @@ tasks:
 commands:
   'cleanup:remote': NpmPrune
 
+options:
+  hooks:
+    - PackageJson:
+        scripts:
+          heroku-postbuild:
+            - 'build:remote'
+            - 'release:remote'
+            - 'cleanup:remote'
```

Comment:
**suggestion**: would be good to document these in the plugins' readmes

**thought**: maybe we could autogenerate that documentation 👀

Reply:
let's augment the doc generation script in a separate PR?